packages/astro/src/core/encryption.ts

```diff
@@ -0,0 +1,65 @@
+import { base64, decodeHex, encodeHex } from '@oslojs/encoding';
+
+const ALGORITHM = 'AES-GCM';
+
+export async function createKey() {
+	const key = await crypto.subtle.generateKey(
+		{
+			name: ALGORITHM,
+			length: 256,
+		},
+		true,
+		['encrypt', 'decrypt']
+	);
+	return key;
+}
+
+export async function importKey(bytes: Uint8Array) {
+  const key = await crypto.subtle.importKey('raw', bytes, ALGORITHM, true, ['encrypt', 'decrypt']);
+  return key;
+}
+
+export async function encodeKey(key: CryptoKey) {
+	const exported = await crypto.subtle.exportKey('raw', key);
+	const encodedKey = base64.encode(new Uint8Array(exported));
+	return encodedKey;
+}
+
+export async function decodeKey(encoded: string) {
+  const bytes = base64.decode(encoded);
+	return crypto.subtle.importKey('raw', bytes, ALGORITHM, true, ['encrypt', 'decrypt']);
+}
+
+const encoder = new TextEncoder();
+const decoder = new TextDecoder();
+const IV_LENGTH = 24;
```

Comment:
Let's document this constant please. I don't know what a `IV length` is in this context.